dev-qt/qtbase/qtbase-6.7.2-r1.ebuild

```diff
@@ -12,7 +12,7 @@ if [[ ${QT6_BUILD_TYPE} == release ]]; then
 fi
 
 declare -A QT6_IUSE=(
-	[global]="+ssl +udev zstd"
+	[global]="+ssl +udev zstd systemd syslog"
 	[core]="icu"
```

Comment:
fwiw `[global]` is only for flags from the top level configure.cmake, and these are from src/corelib/configure.cmake so they should be in `[core]` (also should be alphabetically sorted).

Reply:
Done